README.md

```diff
@@ -30,13 +30,42 @@ opticks-t
 
 ### Docker
 
-Here are basic instructions on how to build and run the `esi-opticks` container using Docker:
+* Create a personal access token with privilege to download packages from github package registry
```

Comment:
Is it really necessary to use a token to pull from a public ghcr.io repository ?

Reply:
The wording in their documentation is kind of vague.  It [documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry) says:
```
You need an access token to publish, install, and delete private, internal, and public packages.
```
I am leaving it in there in case it's needed.